src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -228,13 +228,10 @@ namespace {
 using TaskHandle = ::grpc_event_engine::experimental::EventEngine::TaskHandle;
 
 grpc_core::CallTracerInterface* CallTracerIfSampled(grpc_chttp2_stream* s) {
-  if (s->context == nullptr || !grpc_core::IsTraceRecordCallopsEnabled()) {
```

Comment:
Why was this previously checking `s->context == nullptr`, and why is that check no longer necessary?

Reply:
we don't get context until the first op, but we get an arena at stream creation time